packages/commonwealth/client/scripts/views/components/react_quill_editor/quill_formatted_text.tsx

```diff
@@ -52,7 +52,7 @@ export const QuillFormattedText = ({
   const truncatedDoc: DeltaStatic = useMemo(() => {
     if (isTruncated) {
       return {
-        ops: [...doc.ops.slice(0, cutoffLines)],
+        ops: [...doc!.ops!.slice(0, cutoffLines)],
```

Comment:
Nit: this can actually be null in [this](https://github.com/hicommonwealth/commonwealth/blob/9885315df5941a3a57c82abccb911f4471ad587d/packages/commonwealth/client/scripts/views/components/react_quill_editor/quill_renderer.tsx#L52-L58) case. Maybe we could add `[...(doc?.ops || [ ]).slice(0, cutoffLines)]`